docs/guide/persistence_cache.md

```diff
@@ -59,9 +59,9 @@ The underlying cache system is exposed as an `ibexa.cache_pool` service, and can
 ### Configuration
 
 By default, configuration uses the `cache.tagaware.filesystem` service to store cache files.
-The service is defined in `bin/config/cache_pool/cache.tagaware.filesystem.yml`
+The service is defined in `config/packages/cache_pool/cache.tagaware.filesystem.yaml`
 to use [FilesystemTagAwareAdapter](https://github.com/ibexa/recipes/blob/master/ibexa/oss/4.0.x-dev/config/packages/cache_pool/cache.tagaware.filesystem.yaml#L8).
-This service is loaded through `bin/config/env/generic.php`.
+This service is loaded through `config/env/generic.php`.
```

Comment:
*/env/generic.php & */env/platformsh.php doesn't seem to exist since 3.3

Reply:
[.env](https://symfony.com/doc/5.4/configuration.html#config-dot-env) and [vendor/platformsh/symfonyflex-bridge/platformsh-flex-env.php](https://github.com/platformsh/symfonyflex-bridge/blob/master/platformsh-flex-env.php) are now doing this part on both 3.3 and 4.x

Reply:
See 6b49a79bd2e446e60d2743206329d5b9b415a558